api/src/pcapi/routes/backoffice/gdpr_user_extract/blueprint.py

```diff
@@ -0,0 +1,52 @@
+from flask import render_template
+import sqlalchemy as sa
+
+from pcapi.core.permissions import models as perm_models
+from pcapi.core.users import models as users_models
+from pcapi.models.feature import FeatureToggle
+from pcapi.routes.backoffice import utils
+
+
+gdpr_extract_blueprint = utils.child_backoffice_blueprint(
+    "gdpr_extract",
+    __name__,
+    url_prefix="/gdpr_extract",
```

Comment:
Regarde l’existant pour rester homogène. 

```suggestion
    url_prefix="/gdpr-extract",
```